ember-cli-build.js

```diff
@@ -10,18 +10,18 @@ module.exports = function(defaults) {
       enableExperimentalBuildTimeTransform: true,
       icons: {
         'free-solid-svg-icons': [
-          'faCoffee',
-          'faMagic',
-          'faCircle',
-          'faSquare',
-          'faHome',
-          'faInfo',
-          'faBook',
-          'faPencilAlt',
-          'faCog',
-          'faSpinner',
-          'faCheckSquare',
-          'faSync'
+          'coffee',
+          'magic',
+          'circle',
+          'square',
+          'home',
+          'info',
+          'book',
+          'pencilAlt',
```

Comment:
are these still meant to be `camelCase` instead of `pencil-alt`? I think the names that are passed to this config and to the `fa-icon` component should exactly match what's on the fontawesome website.

Reply:
I have no objection to this. Consistency is awesome.  Just want to check with @robmadole that kebab case is the correct way to go here and what the website will say for 5.1.

Should we then change the components to use this case when referencing an icon?
`{{fa-icon 'pencil-alt'}}`? So it is consistent everywhere?

Reply:
I think @bgentry has the tiger by the tail on this one. It should be `pencil-alt`. If we are going to deviate from doing `faPencilAlt` (which I'm :+1: for, we get to hide the underlying plumbing for this in the Ember component) we need to have it match fontawewome.com/icons.